torchvision/datasets/fgvc_aircraft.py

```diff
@@ -112,3 +114,21 @@ def _download(self) -> None:
 
     def _check_exists(self) -> bool:
         return os.path.exists(self._data_path) and os.path.isdir(self._data_path)
+
+
+def check_if_warning_was_printed():
+    """
+    The samples are organized in sequential order in terms of labels (i.e. 0, ..., 99).
+    Print a one-time warning for the user that in order to avoid the probable bias and performace degragation, the dataloader needs to be shuffled.
+    Check if a warning was already printed, and if not, print it. Saves a boolean file in the torchvision cache directory.
+    """
+    cache_dir = os.path.join(os.path.expanduser('~'), '.cache', 'torch', 'hub', "FGVCAircraft")
+    boolean_file_path = os.path.join(cache_dir, "FGVCAircraft_shuffle_warning.txt")
+
+    if not os.path.exists(boolean_file_path):
+        print("Important: The samples in this dataset are organized sequentially by class.")
+        print("To ensure unbiased performance, it is recommended to shuffle the dataset before use.")
+        print("To do so, you can set the 'shuffle' argument in the dataloader to 'True'.")
```

Comment:
thanks for the PR @EyalMichaeli . As detailed in https://github.com/pytorch/vision/issues/7818#issuecomment-1672786974, we won't be able to merge this as we don't plan on raising warnings for unshuffled datasets.

Just a minor note, hoping that helps: for warnings we don't use `print`, we use `warnings.warn` which can be filtered in specific ways and also avoids warning twice (thus removing the need for the boolean file you created).